taxi/tests/test_views.py

```diff
@@ -0,0 +1,63 @@
+from django.test import Client
+from django.urls import reverse
+import pytest
+
+
+from taxi.models import Manufacturer, Car, Driver
+
+
+@pytest.mark.parametrize(
+    "url",
+    [
+        pytest.param(
+            reverse("taxi:manufacturer-list"),
+            id="manufacturer list"
+        ),
+        pytest.param(
+            reverse("taxi:manufacturer-create"),
+            id="manufacturer create"
+        ),
+        pytest.param(reverse("taxi:car-list"), id="car list"),
+        pytest.param(reverse("taxi:car-create"), id="car create"),
+        pytest.param(reverse("taxi:car-list"), id="car list"),
+        pytest.param(reverse("taxi:driver-list"), id="driver list"),
+        pytest.param(reverse("taxi:driver-create"), id="driver create"),
+
+    ]
+)
+def test_login_required(url):
+    client = Client()
+    res = client.get(url)
+    assert res.status_code != 200
```

Comment:
Instead of checking res.status_code != 200, consider using assert res.status_code == 302 to explicitly check for a redirect.